com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -781,9 +783,10 @@ public void SetState(Vector3? posIn = null, Quaternion? rotIn = null, Vector3? s
             }
             else
             {
-                transform.position = pos;
-                transform.rotation = rot;
-                transform.localScale = scale;
+                m_Transform.position = pos;
+                m_Transform.rotation = rot;
+                m_Transform.localScale = scale;
+                m_LocalAuthoritativeNetworkState.IsTeleportingNextFrame = true;
```

Comment:
Should this not be `m_LocalAuthoritativeNetworkState.IsTeleportingNextFrame = shouldGhostsInterpolate;`